### PR TITLE
prettify `getting-started/index.mdx` and `guides/graphql-server-apollo-yoga.mdx`

### PR DESCRIPTION
**Pull request by B2o5T**
https://graphql-codegen.com/docs/getting-started
vs
https://graphql-code-generator-git-improve-docs-theguild.vercel.app/docs/getting-started

https://graphql-codegen.com/docs/guides/graphql-server-apollo-yoga
vs
https://graphql-code-generator-git-improve-docs-theguild.vercel.app/docs/guides/graphql-server-apollo-yoga

### CODE REVIEW

File: website/docs/getting-started/index.mdx [hunk 1]
```diff
@@ -76,22 +73,22 @@ interface PostQuery {
   }[]
 }
 
-const postsQueryDocument = `
+const postsQueryDocument = /* GraphQL */ `
```
Comment (B2o5T) [inline website/docs/getting-started/index.mdx:76]: adding `/* GraphQL */` comments helps prettier understand that this is not string but graphql document, so he will prettify

File: website/docs/getting-started/index.mdx [hunk 2]
```diff
@@ -157,36 +149,38 @@ const Posts = () => {
   </div>
 </template>
 
-<script>
+<script lang="ts">
```
Comment (B2o5T) [inline website/docs/getting-started/index.mdx:152]: i think lang was missing, after adding this prettier prettified content inside `script` tag

File: website/docs/getting-started/index.mdx [hunk 3]
```diff
@@ -246,27 +242,27 @@ class PostsComponent implements OnInit, OnDestroy {
...
       }
     }
   `
 
-  const posts = query(postsQueryDocument);
+  const posts = query(postsQueryDocument)
 </script>
 
 <ul>
-    { /* UI */ }
```
Comment (B2o5T) [inline website/docs/getting-started/index.mdx:269]: @jycouet does it valid syntax in svelte? I think comment must be `<!-- UI -->`

File: website/docs/getting-started/index.mdx [hunk 4]
```diff
@@ -476,9 +459,9 @@ server.listen().then(({ url }) => {
 import { createServer } from '@graphql-yoga/node'
 import { readFileSync } from 'fs'
 
-import { Resolvers } from './resolvers-types';
+import { Resolvers } from './resolvers-types'
 
-const typeDefs = readFileSync('./schema.graphql').toString('utf-8')
+const typeDefs = readFileSync('./schema.graphql', 'utf8')
```
Comment (B2o5T) [inline website/docs/getting-started/index.mdx:464]: `.toString('utf-8')` is redundant, `utf8` can be passed as 2nd argument to `readFileSync()`

File: website/docs/getting-started/index.mdx [hunk 5]
```diff
@@ -540,6 +522,6 @@ Then, you can either read a guide or go over [the list of available plugins](/pl
 If you are experiencing any issues, you can reach us via the following channels:
 
 - Found a bug? [report it in our GitHub repo](https://github.com/dotansimha/graphql-code-generator)
-- Need help or have a question? You can use the live chat box in the corner of the screen, [ask it in our GitHub Discussions page](https://github.com/dotansimha/graphql-code-generator/discussions) or [reach out to us directly in our Discord](http://bit.ly/guild-chat).
+- Need help or have a question? You can use the live chat box in the corner of the screen, [ask it in our GitHub Discussions page](https://github.com/dotansimha/graphql-code-generator/discussions) or [reach out to us directly in our Discord](http://bit.ly/guild-chat)
```
Comment (B2o5T) [inline website/docs/getting-started/index.mdx:525]: no need dot as it's list

File: website/docs/guides/graphql-server-apollo-yoga.mdx [hunk 6]
```diff
@@ -124,7 +125,7 @@ Assuming that, as recommended, your `package.json` has the following script:
 ```json
 {
   "scripts": {
-    "generate": "graphql-codegen",
```
Comment (B2o5T) [inline website/docs/guides/graphql-server-apollo-yoga.mdx:127]: no need comma as it's json

File: website/docs/getting-started/index.mdx (hunk 3 above)
Comment (jycouet) [reply website/docs/getting-started/index.mdx:269]: You are right ` { /* UI */ }` will not work.
If you want a comment in your HTML, you put HTML comment tag like: `<!-- UI -->`

Comment (jycouet) [reply website/docs/getting-started/index.mdx:269]: While looking at this guide, could we use `KitQL` instead of `Svelte-Apollo` in the example 😇?

Comment (B2o5T) [reply website/docs/getting-started/index.mdx:269]: @dotansimha what do you think?

Comment (charlypoly) [reply website/docs/getting-started/index.mdx:269]: We can do both but we have to keep in mind that guides are here to help most people get started with popular stacks

**Review by charlypoly — APPROVED** [review]

File: website/docs/getting-started/index.mdx (hunk 3 above)
Comment (jycouet) [reply website/docs/getting-started/index.mdx:269]: FYI:
https://github.com/timhall/svelte-apollo => It's without code gen
https://github.com/ticruz38/graphql-codegen-svelte-apollo => is the codegen for 👆 svelte-apollo

Adding in the list will probably not fit.

File: website/docs/getting-started/index.mdx [hunk 7]
```diff
@@ -403,27 +393,21 @@ export class PostsComponent {
...
 
-  - **up-to-date typings**
-
-
-  - **autocompletion on all queries, mutations and, subscription variables and results**
-
-
-  - **less boilerplate** (thanks to full code generation such as React hooks generation)
-
+- **autocompletion on all queries, mutations and, subscription variables and results**
```
Comment (jycouet) [inline website/docs/getting-started/index.mdx:398]: Actually not true in Svelte.
Need https://github.com/ardatan/graphql-tools/pull/4349 and a PR (that I will do now on vscode)

**Review by jycouet — COMMENTED** [review]
Maybe we leave the svelte update on the side to not mix with this PR?

**Comment by jycouet** [comment]
Nice one 👌

**Pull request merged**